tests/test_cmd.sh

```diff
@@ -16,98 +16,89 @@
 #
 # tests for command lines
 
+# Very verbose but useful for debugging.
 set -ex
 
 if [[ "$#" -ge 1 ]]; then
   # eval so that the passed in directory can contain variables.
   BINARY_DIR="$(eval echo "$1")"
 else
-  # assume "tests" is the current directory
-  BINARY_DIR=..
+  # Assume "tests" is the current directory.
+  BINARY_DIR="$(pwd)/.."
 fi
 if [[ "$#" -ge 2 ]]; then
   TESTDATA_DIR="$(eval echo "$2")"
 else
-  TESTDATA_DIR=./data
+  TESTDATA_DIR="$(pwd)/data"
+fi
+if [[ "$#" -ge 3 ]]; then
+  TMP_DIR="$(eval echo "$3")"
+else
+  TMP_DIR=/tmp
```

Comment:
Yannis: Thank you for the refactoring. This pull request doesn't address the output file name collision if someone checks out two libavif trees on the same computer, build them, and run ctest at the same time. Correct?

Reply:
Correct. https://github.com/AOMediaCodec/libavif/pull/1106 should take care of that.